config/.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
 	"singleQuote": true,
 	"trailingComma": "all",
-	"useTabs": true
+	"useTabs": true,
+	"arrowParens": "avoid"
```

Comment:
arrowParens: `"avoid"` or `"always"`? :-)
https://prettier.io/docs/en/options.html#arrow-function-parentheses

Probably `"avoid"` is closer to most existing usage, so yes `"avoid"` is probably better.

Reply:
I took the suggestion to use `avoid` from the bottom of https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-arrowparens-to-always-7430httpsgithubcomprettierprettierpull7430-by-kachkaevhttpsgithubcomkachkaev

> You are encouraged to use the new default value, but if the old behavior is still preferred, please configure Prettier with `{ "arrowParens": "avoid" }`.